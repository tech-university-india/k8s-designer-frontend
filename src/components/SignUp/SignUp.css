.sign-up{
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  height: 100vh;
  background-color: #f5f5f5;
}

.sign-up > h1{
  font-size: 2.5rem;
  font-weight: 500;
  margin-bottom: 30px;
}
.sign-up-form{
  display: flex;
  flex-direction: column;
  align-items: center;
  margin: 0 100px;
  height: 40%;
}

.sign-up-form > input{
  width: 250px;
  height: 55px;
  margin: 10px 0;
  padding: 0 10px;
  border: 2px solid #ccc;
  border-radius: 5px;
}

.sign-up-form >input ::placeholder{
  color: black;
}

.sign-up-button{
  width: 250px;
  height: 55px;
  padding: 0 10px;
  border-radius: 5px;
  background-color: #1034A6;
  color: white;
  font-weight: 500;
  cursor: pointer;
}