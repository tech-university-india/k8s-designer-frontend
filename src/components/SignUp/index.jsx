import React from 'react';
import './SignUp.css';
const SignUp = () => {
  return (
    <div className='sign-up'>
      <h1>SIGN UP</h1>
      <div className='sign-up-form'>
        <input type='text' placeholder='Full Name' />
        <input type='email' placeholder='Email' />
        <input type='password' placeholder='Password' />
        <input type='password' placeholder='Confirm Password' />
      </div>
      <button className='sign-up-button'>SIGN UP</button>
    </div>
  );
};

export default SignUp;