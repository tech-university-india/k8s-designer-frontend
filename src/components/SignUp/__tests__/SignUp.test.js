import React from 'react';
import { render } from '@testing-library/react';
import SignUp from '../index';

describe('Sign up', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<SignUp/>);
    expect(asFragment()).toMatchSnapshot();
  });
});