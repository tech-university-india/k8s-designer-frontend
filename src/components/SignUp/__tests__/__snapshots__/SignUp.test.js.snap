// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sign up should render correctly 1`] = `
<DocumentFragment>
  <div
    class="sign-up"
  >
    <h1>
      SIGN UP
    </h1>
    <div
      class="sign-up-form"
    >
      <input
        placeholder="Full Name"
        type="text"
      />
      <input
        placeholder="Email"
        type="email"
      />
      <input
        placeholder="Password"
        type="password"
      />
      <input
        placeholder="Confirm Password"
        type="password"
      />
    </div>
    <button
      class="sign-up-button"
    >
      SIGN UP
    </button>
  </div>
</DocumentFragment>
`;
